lib/spreedly/gateway_class.rb

```diff
@@ -8,6 +8,7 @@ class GatewayClass
           :supports_void, :supports_verify, :supports_reference_purchase,
           :supports_purchase_via_preauthorization, :supports_offsite_purchase,
           :supports_offsite_authorize, :supports_3dsecure_purchase, :supports_3dsecure_authorize,
+          :supports_3dsecure_2_purchase, :supports_3dsecure_2_authorize,
```

Comment:
Should we add these new characteristics to `GatewayOptionsTest#test_characteristics`?

Reply:
ha, probably. But there is a test smell there -- The tests didn't fail when I ran them 🤣

Reply:
Updated, have a look now :)